src/Templates/AdminPanel/Bit.AdminPanel/Web/Pages/Dashboard/ProductsCountPerCategotyWidget.razor.cs

```diff
@@ -28,20 +36,11 @@ private async Task GetData()
 
             var data = await stateService.GetValue($"{nameof(AnalyticsPage)}-{nameof(ProductsCountPerCategotyWidget)}", async () => await httpClient.GetFromJsonAsync($"Dashboard/GetProductsCountPerCategotyStats", AppJsonContext.Default.ListProductsCountPerCategoryDto));
 
-            Series = new[] {
-                new ColumnSeries<int>()
-                {
-                    Name = "",
-                    Values = data.Select(d => d.ProductCount).ToArray()
-                }
-            };
-
-            XAxis = new[]{
-                new Axis
-                {
-                    Labels = data.Select(d => d.CategoryName).ToArray(),
-                }
-            };
+            BitChartBarDataset<int> chartDataSet = new BitChartBarDataset<int>();
+            chartDataSet.AddRange(data!.Select(d => d.ProductCount));
+            chartDataSet.BackgroundColor = data.Select(d => d.CategoryColor).ToArray();
+            _config!.Data.Datasets.Add(chartDataSet);
```

Comment:
again, I don't like `!`, please find a better solution (for example don't define the config nullable)